DESCRIPTION

```diff
@@ -80,18 +81,20 @@ Imports:
     doParallel,
     parallel,
     DBI,
-    RSQLite
+    RSQLite,
+    tidyr
```

Comment:
It looks like tidyr was already listed above, on new line 58?
```suggestion
    RSQLite
```

Reply:
ah--good call!

await my forthcoming linting and styling PR in which we shall alphabetize the list of imports